app/assets/stylesheets/hyrax/_accessibility.scss

```diff
@@ -0,0 +1,31 @@
+// This stylesheet is a place to apply styling changes for improving the
+// accessibility of Hyrax vanilla
+
+// Testing: WAVE evaluation tool: http://wave.webaim.org/
+// Chrome has a WAVE extension that you can toggle on/off for testing pages
+
+// Bootsrap upto v4.0 has poor styling for color-contrast that causes the WAVE
+// tool to throw contrast errors
+// Read more: https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast
+// These changes meet WCAG AA and AAA compliance standard
+// I'm passing values to bootstrap variables as listed here:
+// https://getbootstrap.com/docs/3.3/customize/#less-variables
+// * Darkens the text for active breadcrumb on every page. Meets WCAG AA and AAA
+// * Darkens the text for disabled pagination buttons on every page.
+$breadcrumb-active-color: #4c4c4c;
+$pagination-disabled-color: #4c4c4c;
+// These variables are only applied if bootstrap is imported after they are set
+// So I'm adding an explicit import here
+@import 'bootstrap';
```

Comment:
We don't want to re-import bootstrap or we'll have two copies of bootstrap in the application.  Instead move your variables to https://github.com/samvera/hyrax/blob/master/app/assets/stylesheets/_bootstrap-default-overrides.scss

Reply:
Hmm, interesting. 
So, move only these 2 variables to _bootstrap-default-overrides.scss  and leave select2-default here?

Reply:
sure.